docker.md

```diff
@@ -0,0 +1,49 @@
+# Docker
+
+**Prerequisite knowledge**: server, infrastructure, virtualization, bash
+
+## Intro
+Docker is used to run software packages called "containers".
+Containers are isolated from each other and bundle their own application, tools, libraries and configuration files;
+they can communicate with each other through well-defined channels.
+All containers are run by a single operating system kernel and are thus more lightweight than virtual machines.
+Containers are created from "images" that specify their precise contents.
+Images are often created by combining and modifying standard images downloaded from public repositories.
+[[Source](https://en.wikipedia.org/wiki/Docker_(software\))]
+
+## Classification / acceptance questions
+
+### Apprentice aka _I have a container running_
+* I can setup Docker on my system
+* I can run an image from a public repository
+
+### Intermediate aka _I have my app running in a container_
+* I know what a Docker image and container are
+* I can define a container with Dockerfile, use it to build the image and run it
+* I can map a container port to a different external port
+* I can publish an image to a repository
```

Comment:
You mentioned mapping ports, maybe you can add a point about mounting a volumen and what is a difference between volumen and using `ADD __file__` in Dockerfile.